CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [0ver](https://0ver.org).
 
 - Configuration for OpenMetrics Encoding in metrics HTTP endpoint. Enabled by default but can be disable using `disable_openmetrics_encoding: true`.
 - Worker pool for projects polling: set `maximum_projects_poller_workers` with an integer value to control parallelism (defaults to `runtime.GOMAXPROCS(0)`)  
+- Augmented `skip_tls_verify` with `skip_health_check` additional parameter to drive the behaviour of checking healthiness of target service 
```

Comment:
well initially, the same bool was used for adding the `TransportConfig` option to the http client _and_ starting the health check.
I thought they are separate concepts:
- skip_tls_verify: removes the TLS verification in health_check_url
- skip_health_check: do not do health check at all

So even if you want to use health check, but have a wrong TLS setup (slef-signed cert is the most common case I can think of) you can still perform health checks on the target service.

Reply:
yes I realized that it could be confusing. However `skip_tls_verify` main purpose is not to disable the TLS verification in the health check but in the actual API calls. I abused it a bit as the library used for the healthcheck did not support (I am not sure if it is still the case) disabling TLS verification.

Reply:
I'd keep it as it is if it's ok for you

Reply:
yeah no worries 👍